herd/tests/instructions/AArch64/A230.litmus

```diff
@@ -0,0 +1,9 @@
+AArch64 A230
```

Comment:
Hi Luc, I have made these changes, however the test currently fails due to a 'Contradiction on reg 0:X2: x vs. x+8` raised by `herd/mem`. I am looking into why this is the case.

Reply:
> Hi Luc, I have made these changes, however the test currently fails due to a 'Contradiction on reg 0:X2: x vs. x+8`raised by`herd/mem`. I am looking into why this is the case.

Reply:
My commit 66c6500b4a7d52b4d2a214f50dfb75455bc094d6  (built on top of this PR) solves this problem by fixing function `kill_regs`. Pleas have a look. It also fixes the dependencies in `-variant kvm`. Producing acceptable dependencies is tricky this is why I have written the commit. I do think that our best option is to intergrate this commit to the PR #503.

Reply:
BTW, looking again at commit, function `write_mem_postindexed` seems to be dead code. I could correct it, but this would change the commit identity and link.